1. values added: 20
2. final result: 20
3. values added: 20
4. error because result is declared with let and then called outside it's block.  Variables declared with let don't work outside the block they are declared in
5. error, result is declared with const and then reassigned, which you can't do
6. error because of the first error